lib/stitches/allowlist_middleware.rb

```diff
@@ -24,5 +23,20 @@ def do_call(env)
       raise 'subclass must implement'
     end
 
+    def configuration
+      @configuration || Stitches.configuration
+    end
+
+  private
+
+    def allowlist_regex
+      regex = @except || configuration.allowlist_regexp
+
+      unless regex.nil? || regex.is_a?(Regexp)
```

Comment:
Shouldn't the second half of this statement be `!regex.is_a?(Regexp)` ?

Reply:
Ah nvm - I see this is just copied from the old code. Grr!

Reply:
Since you've taken the time to refactor this middleware a bit, this check would read better to me as:
```
if !regex.nil? && !regex.is_a?(Regexp)
```